interface/therapy_groups/therapy_groups_models/group_statuses_model.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
What are your thoughts on giving these classes a namespace (maybe just something like 'Group') to prevent conflicts down the road when all classes are autoloaded.